webapp/WEB-INF/templates/admin/plugins/appointment/entries/all_entry_commons.html

```diff
@@ -0,0 +1,12 @@
+<#macro anonymPattern>
+<script type="text/javascript">
+function changeVisilityAnonymizePattern(){
```

Comment:
its better to use events like 
$('#anonymizable' ).change ( ) {  
   // if $(this).checked ...